### DISCUSSION TITLE
Can Hanko support multi-tenant teams?

### DISCUSSION THREAD

**Discussion by Zyles** [teamhanko/hanko#1975] [Q&A]
Can Hanko do multi tenant organizations with teams where a user can be member of multiple teams with different roles?

```
Org 1
- Team 1
    - User ABC (admin role)
    - User CCC
- Team 2
    - User ABC (editor role)

Org 2
- Team 1
- Team 2
```

**Comment by FlxMgdnz** [comment]
Not possible as of v2.1, but we have started working on an "Organizations" feature that will give you exactly that.

**Reply by Zyles** [reply]
> Not possible as of v2.1, but we have started working on an "Organizations" feature that will give you exactly that.

Okay I hope you make sure to add at least teams within orgs. Most other solutions have failed at this and just make org level which requires multiple separate organizations for one company. And this requires a super admin to have multiple logins to multiple orgs.

**Reply by FlxMgdnz** [reply]
Thanks for the input, super interesting. So far we have not thought about teams inside of organizations. What would be the real life use case for teams?

**Reply by Zyles** [reply]
Imagine a semi-large to large company. For example you have Microsoft, they have teams that work on Office 365, Windows or other small apps. Should they create an org for every single app inside their organization? If you work in support department you naturally do not need logins to handle things in development departments.

A team member might be the owner of app A but just be a contributor of app B, they should not have the same credentials across all teams.

And there can be a team manager that is in charge of multiple teams too.
